content/influxdb/cloud/write-data/developer-tools/api.md

```diff
@@ -15,7 +15,7 @@ Use the `POST` request method and include the following in your request:
 |:-----------          |:----------                                               |
 | Organization         | Use the `org` query parameter in your request URL.       |
 | Bucket               | Use the `bucket` query parameter in your request URL.    |
-| Precision            | Use the [`precision`](/influxdb/cloud/write-data/#timestamp-precision) query parameter in your request URL. Default is `ns` |
+| Precision            | Use the [`precision`](/influxdb/cloud/tools/influxql-shell/#precision) query parameter in your request URL. Default is `ns` |
```

Comment:
This isn't the correct link. This points to the precision setting of the InfluxQL shell, which would seem weird in this context. This link should actually point to https://docs.influxdata.com/influxdb/cloud/write-data/developer-tools/line-protocol/#timestamp-precision

```suggestion
| Precision            | Use the [`precision`](/influxdb/cloud/write-data/developer-tools/line-protocol/#timestamp-precision) query parameter in your request URL. Default is `ns` |
```